src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -161,7 +179,10 @@ struct EvolutionMetavars {
       Actions::ComputeTimeDerivative<
           evolution::dg::ConservativeDuDt<Burgers::System, dg_formulation>>,
       evolution::Actions::AddMeshVelocitySourceTerms,
-      dg::Actions::ImposeDirichletBoundaryConditions<EvolutionMetavars>,
+      tmpl::conditional_t<
+          evolution::is_analytic_solution_v<initial_data>,
+          dg::Actions::ImposeDirichletBoundaryConditions<EvolutionMetavars>,
+          tmpl::list<>>,
```

Comment:
What BCs do you use for analytic data...none? How does that work?

Reply:
That's a good question.

Burgers, like NewtonianEuler, only has periodic analytic data (so far), so no BCs are needed for these evolutions (so far).

I'm not sure what happens for the GrMhd analytic data that are not periodic (e.g., the disk initial conditions). Do we just do nothing at the boundaries there?

Reply:
We haven't run those yet since we can't even get the analytic solution disk to work.

We currently can't handle BCs at all so I'm not worried about this.

Reply:
Ah, I didn't realize we hadn't run those particular solutions. Makes sense though 🙂